dub.sdl

```diff
@@ -10,7 +10,6 @@ dependency "libdparse" version="~>0.7.0"
 
 configuration "application" {
 	targetType "executable"
-	mainSourceFile "source/app.d"
```

Comment:
Can you add a comment `// Temporarily disabled to work around dlang/dub#1407`, as a reminder to add it back later?

Reply:
I see. I will add a comment.